lib/CleantalkSP/SpbctWP/VulnerabilityAlarm/VulnerabilityAlarmService.php

```diff
@@ -15,10 +15,53 @@ class VulnerabilityAlarmService
     public static function updateVulnerabilitiesLibrary()
     {
         $modules = self::getModules();
+        self::setSummaryLogPluginsAndThemes($modules);
```

Comment:
The summary saving must be acted after writing report method call.

Reply:
https://github.com/CleanTalk/security-malware-firewall/pull/344/commits/822b8f7ed03c18e06f7fc0942865d21559052545